.ci/travis_script.sh

```diff
@@ -5,7 +5,7 @@ set -e -x
 if [[ -n ${TOXENV} ]]; then
   tox -e ${TOXENV}
 elif [[ "${BIGCHAINDB_DATABASE_BACKEND}" == mongodb && \
-    "${BIGCHAINDB_DATABASE_SSL}" == false ]]; then
+    -z "${BIGCHAINDB_DATABASE_SSL}" ]]; then
```

Comment:
So `-z "${BIGCHAINDB_DATABASE_SSL}"` will be true if `BIGCHAINDB_DATABASE_SSL` is not set?

Bash is so strange. (This is called Bash bashing.)